doc/electrs-rest-btcd-1.0.0.yaml

```diff
@@ -0,0 +1,936 @@
+openapi: 3.0.0
+tags:
+  - name: address
+    description: Address Operations
+  - name: block
+    description: Block Operations
+  - name: tx
+    description: Transaction Operations 
+  - name: scripthash
+    description: Script Operations
+  - name: mempool
+    description: unconfirmed Block Operations
+  - name: default
+    description: Everything else
+info:
+  title: Electrs Rest API
+  version: 1.0.0
+  description: |
+    A Rust Implementation of the Electrum Server.
+    These docs are for the Standard Bitcoin Network Only.
+servers: 
+  - description: "Blockstream Public (bitcoin)" 
+    url: "https://blockstream.info/api/"
+  - description: "Blockstream Public (liquid)" 
+    url: "https://blockstream.info/liquid/api/"
+paths:
+  /blocks/tip/hash:
+    get:
+      tags:
+        - block
+      summary: Get the hash of the tip block
```

Comment:
Suggest
"Get the hash of the best tip block"

Reply:
added